pyngrok/ngrok.py

```diff
@@ -180,7 +180,7 @@ def connect(port="80", proto="http", name=None, options=None, pyngrok_config=Non
     tunnel = NgrokTunnel(api_request("{}/api/{}".format(api_url, "tunnels"), method="POST", data=options,
                                      timeout=pyngrok_config.request_timeout))
 
-    if proto == "http" and ("bind_tls" not in options or options["bind_tls"] is not False):
+    if proto == "http" and ("bind_tls" not in options or options["bind_tls"] is False):
```

Comment:
Let's do this instead:

```
 if proto == "http" and not options.get("bind_tls", False):
```

Reply:
> Let's do this instead:
> 
> ```
>  if proto == "http" and not options.get("bind_tls", False):
> ```

PR is merged so can't edit it. I'll create another one

Reply:
No need, I fixed it after the merge and added some tests, it's in `4.1.10`.